src/wp-includes/formatting.php

```diff
@@ -4536,6 +4542,63 @@ function esc_textarea( $text ) {
 	return apply_filters( 'esc_textarea', $safe_text, $text );
 }
 
+/**
+ * Escaping for XML blocks.
+ *
+ * @since 5.5.0
```

Comment:
I would love a more detailed description here that explains why someone would use this function rather than `esc_attr()` or `esc_html()`.

Reply:
We can probably update the docblocks of the other functions too while we're at it. Currently they're a bit bleak...

Reply:
will see what I can come up with that's a) more helpful that what's there and b) shorter than a dissertation :-)